Tidy imports

rastrau opened the pull request:
Imports have been tidied up, removed duplicates and consolidated some redundant and `from ...` imports.

rastrau committed:
Tidy imports

davidoesch commented on swisstopoBatchNmerge.py:
```diff
@@ -105,7 +99,7 @@ def osm_to_decimal(tile_x: Union[int, float], tile_y: Union[int, float], zoom: i
 # file explorer window
 def browseFiles():
```
Diese Funktion wird nicht mehr benötigt, kann ich eigentlich entfernen

davidoesch committed:
add product preview when switching product

-might be slow since internal cache is updated
-not yet in the windwos exe
-added PR 7

davidoesch committed:
Delete map_widget.py

davidoesch reviewed (commented):
please test and review as well the perfromance ona layer switch
